doc/time-series.rst

```diff
@@ -197,11 +197,20 @@ resampling group:
 
    ds.resample(time='6H').reduce(np.mean)
 
-For upsampling, xarray provides four methods: ``asfreq``, ``ffill``, ``bfill``,
-and ``interpolate``. ``interpolate`` extends ``scipy.interpolate.interp1d`` and
-supports all of its schemes. All of these resampling operations work on both
+For upsampling, xarray provides six methods: ``asfreq``, ``ffill``, ``bfill``, ``pad``,
+``nearest`` and ``interpolate``. ``interpolate`` extends ``scipy.interpolate.interp1d``
+and supports all of its schemes. All of these resampling operations work on both
 Dataset and DataArray objects with an arbitrary number of dimensions.
 
+In order to limit the scope of the methods ``ffill``, ``bfill``, ``pad`` and
```

Comment:
Mention that "tolerance" is in coordinate (or label) units, not index units and that the index version is limit

Reply:
The unit for ``tolerance`` is now (a878ebc62825ea0e281960ca2e91348899f34a60) explicitly mentioned in the time-series example. However, I did not comment on the ``index`` method as this has not been implemented for the resample method, yet. The index method seems to be a bigger/other project.